03-install-kernel.sh

```diff
@@ -40,7 +40,7 @@ then
     cd uboot
 
     infecho "Writing bootloader..."
-    dd if=uboot.bin of=$PP_IMAGE bs=1024 seek=8
+    dd if=u-boot-sunxi-with-spl.bin of=$PP_IMAGE bs=1024 seek=8
```

Comment:
This is for the new uploaded uboot which comes from here: https://gitlab.com/fedora-mobile/yoda-uboot